teams/teams-ps/teams/Set-CsTeamsAIPolicy.md

```diff
@@ -0,0 +1,104 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+applicable: Microsoft Teams
+title: Set-CsTeamsAIPolicy
+online version: https://learn.microsoft.com/powershell/module/teams/Set-CsTeamsAIPolicy
+schema: 2.0.0
+author: Andy447
+ms.author: andywang
+---
+
+# Set-CsTeamsAIPolicy
+
+## SYNOPSIS
+
+This cmdlet Set Teams AI policy value for current tenant.
+
+## SYNTAX
+
+```
+Set-CsTeamsAIPolicy -Identity <string> -EnrollVoice <Enabled/Disabled>
+```
+
+## DESCRIPTION
+
+This cmdlet Set Teams AI policy EnrollFace/EnrollVoice value for current tenant. The value can be "Enabled" or "Disabled"
```

Comment:
Please make this description a little more clear. It is a bit ambiguous.

Recommended Edits:
This cmdlet sets the Teams AI policy EnrollFace and EnrollVoice value for the tenant. The values of EnrollFace and EnrollVoice can be set to "Enabled" or "Disabled"